AWS/logging_lambda/crawling_log_lambda.py

```
@@ -0,0 +1,55 @@
+SLACK_WEBHOOK_URL = "https://hooks.slack.com/services/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"
+
+def send_slack_alert(log_data):
+    """Slack Webhook으로 오류 알림 전송"""
+    if not SLACK_WEBHOOK_URL:
+        print("❌ Webhook URL이 설정되지 않았습니다.")
+        return {"status": "failed", "error": "Missing SLACK_WEBHOOK_URL"}
+
+    # Slack 메시지 구성
+    message = (
+        f"🚨 *오류 발생!*\n"
+        f"- 발생 시간: {log_data.get('timestamp', (datetime.utcnow() + timedelta(hours=9)).strftime('%Y-%m-%d %H:%M'))}\n"
```

Ruff (0.8.2)

Undefined name `timedelta`

(F821)